package.json

```diff
@@ -1,8 +1,17 @@
 {
+  "type": "module",
   "dependencies": {
+    "bcrypt": "^5.0.1",
     "express": "^4.18.0",
     "express-session": "^1.17.2",
     "jsdom": "^19.0.0",
-    "mysql2": "^2.3.3"
+    "mongodb": "^4.5.0",
+    "mysql2": "^2.3.3",
```

Comment:
Should remove Jay's old MySQL code, maybe in this PR if it's quick or in another one if not.